tools/pihole_toml_to_markdown.py

```diff
@@ -0,0 +1,174 @@
+import re
+from pathlib import Path
+
+def parse_toml_with_comments(filepath):
+    with open(filepath, encoding='utf-8') as f:
+        lines = f.readlines()
+
+    documentation = ["""---
+description: This file is automatically generated when new code is pushed to the master branch of the FTL Repo. Any proposed changes should be made against https://github.com/pi-hole/ftl/tools/pihole_toml_to_markdown.py
+---
```

Comment:
This string will be used by link-preview generators, e.g.
![image](https://github.com/user-attachments/assets/b234cec9-e412-407f-a526-0ec8ab28539f)

I don't think this is a good fit and should instead just say that we always show the config options of the most recent FTL release here. This sentence here could go towards the end of the page, it is not important and probably nobody would file a PR against the documentation *first*.

Reply:
This can be changed the a Note or just a paragraph, at the end of the file.